src/OmniSharp.MSBuild/OmniSharp.MSBuild.csproj

```diff
@@ -1,8 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net472</TargetFramework>
+    <TargetFrameworks>net472;netcoreapp2.1</TargetFrameworks>
```

Comment:
I am also a bit surprised by the appearance of `netcoreapp2.1` target 😀

Reply:
Me too, but https://www.nuget.org/packages/Microsoft.Build/ supports it, figured we might as well also.